Mods/ArchipelagoTriggers.SC2Mod/Base.SC2Data/LibABFE498B.galaxy

```diff
@@ -7004,6 +7024,7 @@ bool libABFE498B_gt_AP_Triggers_setOptions_Func (bool testConds, bool runActions
     lv_ncoOnlyRun = StringWord(EventChatMessage(false), 13);
     lv_storyLevels = StringWord(EventChatMessage(false), 14);
     lv_enableMorphling = StringWord(EventChatMessage(false), 15);
+    lv_overrideRace = StringWord(EventChatMessage(false), 16);

```

Comment:
Need to reserve that on Client side (make a PR for that, If the value is `0` sent from the client, do not override)

Possibly make it dummy, let the client send always `0` (with comment Player race override)

Reply:
I agree, yeah. That's what I've been doing in my testing, it's just that the actual logic of adding map variations has turned out to be a lot more complicated than I'd expected (and I've been too busy to work on it for a little while). I've converted this PR to a draft so it's clearer that it's not ready to merge yet.

Reply:
screw it, I'm just gonna make a client PR to reserve the 16th word of the string like you suggested, the generator logic isn't going to be ready for a while and I want to put this through before too many conflicts arise

Reply:
https://github.com/Ziktofel/Archipelago/pull/214